README.md

```diff
@@ -347,6 +348,34 @@ ch.consume(q, function cons(msg) {
 });
 ```
 
+### bunnybus
+
+Tracing bunnybus consumers can be done in two methods:
+1. [Auto-tracing](#auto-tracing) using the environment variable.
+2. Calling the SDK.
+
+Calling the SDK is simple, and should be done in your main `js` file where the consumer is being initialized:
+
+```javascript
+const epsagon = require('epsagon-frameworks');
+
+epsagon.init({
+  token: 'epsagon-token',
+  appName: 'app-name-stage',
+  metadataOnly: false,
+});
+```
+
+Tagging traces or setting custom errors can be by:
+
+```javascript
+handler: async (msg, meta, ack, rej, epsagon) => {
+    msg.epsagon.label('key', 'value');
+    msg.epsagon.setError(Error('My custom error'));
+    await ack();
+}
```

Comment:
@ranrib , I just want to make sure our contracts are not broken.

When `subscribe(...)` is invoked and `options.meta === true`, the handler is called with the following:

```javascript
handler: async (msg, meta, ack, rej, requeue) => {
    msg.epsagon.label('key', 'value');
    msg.epsagon.setError(Error('My custom error'));
    await ack();
}
```

When `subscribe(...)` is invoked with out `options.meta` set or if `options.meta === false`, the handler is called with the following which is the normal use case.  Note the `meta`  prop is no longer passed.

```javascript
handler: async (msg, ack, rej, requeue) => {
    msg.epsagon.label('key', 'value');
    msg.epsagon.setError(Error('My custom error'));
    await ack();
}
```

I notice the trailing `epsagon` parameter being available now.  I just want to make sure this parameter is not squating over the `requeue` parameter this is passing in along the same arity.

Reply:
Yes you're right. It gets appended as the last argument, and doesn't override any. I've added some clarifications here

Reply:
@ranrib, The new public interface for the handler(s) registered to `subscribe` in `@tenna-llc/bunnybus@v7.x` is as follows

Example Handler
```javascript
async ({ message, metaData, ack, rej, requeue }) => { }
```

Full Use Example
```javascript
const BunnyBus = require('@tenna-llc/bunnybus');
const bunnybus = new BunnyBus();

await bunnyBus.subscribe({
    queue: 'queue1',
    handlers: {
        'departments.#': async ({ message, metaData, ack, rej, requeue }) => await ack()
    }
});
```